build.gradle

```diff
@@ -0,0 +1,56 @@
+group 'com.kllawesome'
+version '0.1.1'
+buildscript {
+    ext.kotlin_version = '1.1.51'
```

Comment:
Were we going with Kotlin? I'm down to play with/learn that, I just thought folks had talked about running with clojure earlier

Reply:
It's just what I know and no movement has been made to build anything. I'm happy to do something else. This is one reason I put up the preliminary PR since I know others were talking clojure.

Reply:
Cool, yeah glad to have some movement on it! Just have to see what the others think :)